src/contoso-fiber/pom.xml

```diff
@@ -123,6 +125,11 @@
 			<artifactId>spring-boot-starter-aop</artifactId>
 		</dependency>
 
+		<dependency>
```

Comment:
Why is this dependency necessary?

Reply:
StringUtils is used here
https://github.com/Azure/reliable-web-app-pattern-java/blob/b4181e6a857fa76354aebca79d7bf0462e250d7c/src/contoso-fiber/src/main/java/com/contoso/cams/serviceplan/ServicePlanExceptionSimulator.java